src/app/pages/cars/cars/cars-list-table-data-source.ts

```diff
@@ -218,17 +232,34 @@ export class CarsListTableDataSource extends TableDataSource<Car> {
 
   public buildTableRowActions(): TableActionDef[] {
     return [
-      this.editAction
+      this.editAction,
+      this.deleteAction,
     ];
   }
 
+  public canDisplayRowAction(actionDef: TableActionDef, car: Car) {
+    const foundCarUser = car.carUsers.find((carUser) => carUser.user.id === this.centralServerService.getLoggedUser().id);
+    switch (actionDef.id) {
+      case CarButtonAction.DELETE_CAR:
+        return this.isBasic ? foundCarUser.owner : true;
```

Comment:
The basic user can delete his car even if it is not the car owner, in that case only the carUsers will be deleted.